### ISSUE TITLE
Reply ID not found

### ISSUE THREAD

**Issue by outrageousinc-dev** [Voyz/ibind#35]
Hi there!

I'm using ibeam and ibind with the ibkr API. I keep getting this error when placing orders:

```
ibind.support.errors.ExternalBrokerError: IbkrClient: response error Result(data=None, request={'url': 'https://ibkr-gateway.agent.local:5000/v1/api/iserver/reply/7adec048-1dd9-42d6-bcf0-e52698293303', 'json': {'confirmed': True}}) :: 400 :: Bad Request :: 
{
    "error": "reply id not found: '7adec048-1dd9-42d6-bcf0-e52698293303'"
}
```

Any idea why this could be?

in

```
File "/usr/local/lib/python3.11/site-packages/ibind/base/rest_client.py", line 189, in _process_response
raise ExternalBrokerError(f'{self}: response error {result} :: {response.status_code} :: {response.reason} :: {response.text}', status_code=response.status_code) from e
```

Thanks in advance, and have a great day! :)

**Comment by Voyz** [comment]
Hey @outrageousinc-dev thanks for trying out IBeam and IBind and reporting your issue 👍

This error indicates that in the order placement question/answer exchange your client has encountered a situation in which giving an answer to a question was not recognised by IBKR's servers. 

It is quite strange, as this answer ID (what you see after `reply`) is read directly from the question we've received.

Does this happen to you every time you submit an order? Was this only one instance?

**Comment by ChienHsiao** [comment]
I encountered the same situation and I'm not sure if it's related to the fact that I'm running multi-threaded trading.

**Comment by Voyz** [comment]
@ChienHsiao can you provide more info? Multi threaded was fine, but were you submitting multiple orders at the same time?

**Comment by ChienHsiao** [comment]
Yes, I did submit multiple orders at the same time, but after I ensured that only one order was submitted at a time, I no longer received the same error.

**Comment by Voyz** [comment]
Okay, that must be it then - submitting multiple orders at the same time. Which makes sense, the replies get mixed up. I should add this limit to the library itself though, thanks for pointing it out 👍

**Comment by Voyz** [comment]
@ChienHsiao please see `ibind[oauth]==0.1.11-rc1` for a version which only allows submitting one order at a time. Let me know if it works as expected 👍

**Comment by ChienHsiao** [comment]
@Voyz After testing with the paper account, everything is running smoothly. By the way, I really appreciate you providing the ibind and ibeam packages—they've played a huge role in the rapid growth of my business.

**Comment by Voyz** [comment]
Superb @ChienHsiao happy to hear 👏 And thanks for the kind words, I'm happy to hear it helps!

**Comment by weklund** [comment]
I just ran into this issue! :D Perfect timing. I'll get the latest release and see if that resolves it for me too.

**Comment by weklund** [comment]
Sorry I meant I was just referring to my 1st order.  

It's just odd that I would experience this issue on a 2nd order, when I was able to successfully get the fill price on the 1st.  I'll follow up with logs here in a bit.

**Comment by Voyz** [comment]
I'm going to close this issue due to inactivity. Thanks for your contribution and please feel free to request a reopen if you'd like to continue the discussion 👍
